.github/workflows/ci-testing.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous testing & Linting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r .virtualenv.dev-requirements.txt
+        pip install -e .
+    - name: Lint with flake8
```

Comment:
I usually avoid spaces in name tags. Again just personal taste, feel free to ignore it :)

Reply:
Ack, will stick with them for now.